### PR TITLE
Update statutes and code of conduct page

### PR DESCRIPTION
**Pull request by rantahar**
- Official statutes
 - Add a link to the board to the code of conduct page

### CODE REVIEW

File: content/about/governance/_index.md [hunk 1]
```diff
@@ -26,98 +18,188 @@ While purpose of the association is to support research software engineering and
...
-the English translation is equivalent.
+The Finnish statutes below are actually official, but the English translation is
+as faithful as possible.
 
 
 ## Statutes of the Association in English
 
 1. Name and domicile of the association:
-    The name of the association is the Nordic Research Software Engineers Ry and it is domiciled in Helsinki, Finland
+    The name of the association is the Nordic Research Software Engineers ry and it is domiciled in Helsinki, Finland
```
Comment (lucaferranti) [inline content/about/governance/_index.md:51]: here ry is with small r, at the beginning of the page it has a capital r

File: content/about/governance/_index.md [hunk 2]
```diff
@@ -2,20 +2,12 @@
...
->
-> This page is a draft and is subject to change. The association is not yet registered
-> and we are still considering our options.
->
 
 # The Association
 
-The Nordic RSE group is planning to register in Finland as the
-Nordic Research Software Engineers Ry.
+We are registered in Finland as the Nordic Research Software Engineers Ry.
```
Comment (lucaferranti) [inline content/about/governance/_index.md:8]: should `Ry` be uncapitalized here (see comment below)?

File: content/about/governance/_index.md [hunk 3]
```diff
@@ -26,98 +18,188 @@ While purpose of the association is to support research software engineering and
...
+# Board
+
+The association is governed by a board. In practice, the board will take care of
+the dull day-to-day affairs and, for now, function as the code of conduct
+committee.
+
+The current board members are
+ - Jarno Rantaharju (Aalto University), jarno.rantaharju@aalto.fi
+ - Anne Fouilloux (University of Oslo), annefou@geo.uio.no
+ - Luca Ferranti (University of Vaasa)
```
Comment (lucaferranti) [inline content/about/governance/_index.md:34]:
```suggestion
 - Luca Ferranti (University of Vaasa), luca.ferranti@uwasa.fi
```

File: content/about/governance/_index.md [hunk 4]
```diff
@@ -26,98 +18,188 @@ While purpose of the association is to support research software engineering and
...
+ - Rohit Goswami (University of Iceland)
+ - Samantha Wittke (Aalto University, CSC)
+ - Radovan Bast (UiT The Arctic University of Norway)
+ - Thomas Arildsen (Aalborg University)
+ - Patric Holmvall (Uppsala University)
```
Comment (PatricHolmvall) [inline content/about/governance/_index.md:39]:
```suggestion
 - Patric Holmvall (Uppsala University), patric.holmvall@physics.uu.se
```

**Review by PatricHolmvall — APPROVED** [review]

File: content/about/governance/_index.md (hunk 1 above)
Comment (rantahar) [reply content/about/governance/_index.md:51]: It should not be capitalized. I'll check the other places

**Review by lucaferranti — APPROVED** [review]

**Review by bast — APPROVED** [review]

File: content/about/governance/_index.md [hunk 5]
```diff
@@ -26,98 +18,188 @@ While purpose of the association is to support research software engineering and
...
+    The name of the association is the Nordic Research Software Engineers ry and it is domiciled in Helsinki, Finland
 
 2. Purpose and forms of activity:
-    The purpose of the association is to support research software engineering and scientific computing.
+    The purpose of the association is to support research software engineering and computational research.
+
+    To fulfill its purpose, the association
+      - may organize courses, workshops and other similar events
+      - may organize confenrences, seminars and other similar events
+      - may maintain electronic communication channels
```
Comment (ThomasA) [inline content/about/governance/_index.md:59]: conferences

File: content/about/governance/_index.md [hunk 6]
```diff
@@ -26,98 +18,188 @@ While purpose of the association is to support research software engineering and
...
+    The annual meeting decides size of the membership fee and registration fee.
 
 5. Board:
-    The association's affairs are handled by the board, which has a chair and 3-7 other regular members, as well as 0-8 deputies, all of whom are appointed at the annual meeting. The term of office of the Board is the time between the annual meetings.
+    The association's affairs are handled by the board, which has a chair and 3-7 other regular members, as well as 0-8 deputies, all of whom are appointed at the annual meeting. The term of office of the board is the time between the annual meetings.
+
+    The board chooses a vice their from amongst its members, and additionally a secretary, a treasurer and any additional necessary functionaries. Board meetings are convened by the chair, or the vice chair when the chair is unable, when they consider it necessary of at least half of the board members request it. The board has a quorum when at least half of its members, the chair and vice chair included, are present. Votes are resolved by strict majority. In the event of a tie, the chair has the casting vote, except in elections where ties are broken by lot.
```
Comment (ThomasA) [inline content/about/governance/_index.md:83]: Shouldn't "vice their" be vice chair?

Comment (ThomasA) [reply content/about/governance/_index.md:83]: "of at least half of the board members request it" -> ", or at least half of the board members request it"

File: content/about/governance/_index.md [hunk 7]
```diff
@@ -26,98 +18,188 @@ While purpose of the association is to support research software engineering and
...
+
+    The meetings are held in English.
+
+10. Annual meeting:
+    The following issues will be discussed at the annual meeting
+    1. opening the meeting
+    2. choosing a chair, secretary, two auditors of the minutes and when necessary two vote counters
+    3. determining quorum and legality of the meeting
+    4. approving the agenda for the meeting
+    5. the associations financial statements, annual report and the statements of the auditors are presented
```
Comment (ThomasA) [inline content/about/governance/_index.md:109]: association's

**Review by ThomasA — CHANGES_REQUESTED** [review]
I came across a couple of typos I think should be corrected.

**Comment by bast** [comment]
These corrections got fixed in #347 (I read the email notifications out of order).

**Pull request merged**